deploy/charts/harvester/dependency_charts/harvester-network-controller/templates/daemonset.yaml

```diff
@@ -52,7 +61,9 @@ spec:
       volumes:
         - hostPath:
             path: /dev
+            type: Directory
           name: dev
         - hostPath:
             path: /lib/modules
-          name: modules
+            type: Directory
```

Comment:
This check may fail the installation in app mode

Reply:
May be